src/pretalx/eventyay_common/views/auth.py

```diff
@@ -39,8 +43,11 @@ def oauth2_login_view(request, *args, **kwargs):
 
 
 def oauth2_callback(request):
+    sso_provider = SocialApp.objects.filter(
+        provider=settings.EVENTYAY_SSO_PROVIDER
+    ).first()
     oauth2_session = OAuth2Session(
-        settings.OAUTH2_PROVIDER["CLIENT_ID"],
+        sso_provider.client_id,
```

Comment:
`.first()` returns nullable object. `sso_provider` can be `None` and `sso_provider.client_id` will raise error.